wallet/how-to/use-non-evm-networks/starknet/manage-starknet-accounts.md

```diff
@@ -0,0 +1,155 @@
+---
+description: Manage Starknet accounts in MetaMask.
+sidebar_position: 2
+---
+
+# Manage Starknet accounts
+
+## Account creation
```

Comment:
As we discussed during the sync, maybe move this section lower. Also, add a brief intro to this page.